llama_stack/cli/stack/configure.py

```diff
@@ -112,12 +112,6 @@ def _run_stack_configure_cmd(self, args: argparse.Namespace) -> None:
             )
             return
 
-        build_name = docker_image.removeprefix("llamastack-")
-        saved_file = str(builds_dir / f"{build_name}-run.yaml")
-        cprint(
-            f"YAML configuration has been written to {saved_file}. You can now run `llama stack run {saved_file}`",
-            color="green",
-        )
```

Comment:
ok, last one I think 😆

did you intend to re-introduce this msg, as well?

Reply:
Nope, this line is a duplicate to L172 which has caused confusion 😆

Reply:
ah, i see!  thanks!